### ISSUE TITLE
Removed rules not mentioned in v7 changelog

### ISSUE THREAD

**Issue by m-allanson** [stylelint/stylelint#1636]
> Describe the issue. Is it a bug or a feature request (new rule, new option, etc.)?

The v7 changelog doesn't mention that some rules have been removed in v7. Running stylelint v7 with a config that uses a missing rule throws an error:

```shell
Error: Undefined rule "property-unit-blacklist"
```

The error is good, but I'd expect to see the rule name mentioned in the v7 section of the changelog. It mentions that the rule has been deprecated but in the case of `unit-property-blacklist`, it's several versions ago and the rule is only described as `property-*-blacklist`, making it difficult to search for.

I think the changelog should be updated to specifically mention that `property-unit-blacklist` has been removed.

> Which rule, if any, is this issue related to?

I've seen this error for `property-unit-blacklist`, based on [this issue](https://github.com/stylelint/stylelint/issues/1454) and the [v7 changelog](https://github.com/stylelint/stylelint/blob/v7/CHANGELOG.md) it seems that the following rules are affected:

`number-zero-length-no-unit`
`property-unit-blacklist`
`property-unit-whitelist`
`property-value-blacklist`
`property-value-whitelist`

> What stylelint configuration is needed to reproduce this issue?

```json
{
  "rules": {
    "property-unit-blacklist": {
      "line-height": [""]
    }
  }
}
```

> Which version of stylelint are you using?

7.0.0

> How are you running stylelint: CLI, PostCSS plugin, Node API?

CLI with `stylelint 'app/**/*.css'`

> Does your issue relate to non-standard syntax (e.g. SCSS, nesting, etc.)?

No

> What did you expect to happen?

A note in the changelog to say that rule `property-unit-blacklist` has been removed.

> What actually happened (e.g. what warnings or errors you are getting)?

Checking the v7 changelog does not indicate that the rule has been removed.

**Comment by jeddy3** [comment]
My bad. Missing CHANGELOG update in [this PR](https://github.com/stylelint/stylelint/pull/1456). I'll fix now.

**Comment by davidtheclark** [comment]
Fixed in #1456.
